website/src/components/content/CaseStudyPage.tsx

```diff
@@ -98,22 +99,53 @@ export const MediaQuote: React.FunctionComponent<{
 
 export const CaseStudyJumbotron: React.FunctionComponent<{
     customer: string
-    logo: string
+    logo?: string
     className?: string
     color?: keyof typeof COLORS
     children?: React.ReactNode
 }> = ({ customer, logo, className = '', color = 'dark', children }) => (
-    <div className={`jumbotron rounded-0 ${COLORS[color]} ${className}`}>
-        <div className="container text-center pt-3 pb-3">
+    <div className={`jumbotron rounded-0 d-flex flex-column ${COLORS[color]} ${className}`}>
+        <div className="pl-lg-7 pl-sm-0 pt-5 container">
+            {color === 'dark' ? (
+                <BackButtonLight href={'/case-studies'} text="Case Studies" />
+            ) : (
+                <BackButtonBold href={'/case-studies'} text="Case Studies" />
+            )}
+        </div>
+        <div className="container text-md-center">
             {logo && <img className="case-studies__logo my-3" src={logo} alt={customer} />}
-            <span className="case-studies__label d-block mt-1">
-                <span className="sr-only">{customer}</span> case study
-            </span>
+            <h5 className="font-weight-bold mt-1">
+                <span className="sr-only">{customer}</span> Case Study
+            </h5>
             {children}
         </div>
     </div>
 )
 
+export const AuthorBio: React.FunctionComponent<{
+    customer: string
+    image?: string
+    author: string
+    title: string
+    about: string
+}> = ({ customer, image, author, title, about }) => (
+    <div className="d-flex flex-md-row flex-column align-items-center align-items-md-start container-xl py-6 py-md-8">
+        {image && (
+            <div className="col-md-3 text-center text-md-right">
+                <img className="rounded-circle p-1" style={{ border: '3px solid #00A1C7' }} src={image} alt={author} />
+            </div>
+        )}
+        <div className="col-md-3 col-lg-2 col-9 text-center text-md-left">
```

Comment:
I'd suggest changing `col-md-3 col-lg-2` to `col-md-4 col-xl-3` and then removing `col-9`. Since divs are block level, they inherit the parent width by default and `container-xl` is applied to the parent. So this div will span across the width on mobile and adding `col-9` as the default isn't necessary 🙂